packages/relay/tests/lib/eth/eth_call.spec.ts

```diff
@@ -117,21 +118,6 @@ describe('@ethCall Eth Call spec', async function () {
       process.env.ETH_CALL_DEFAULT_TO_CONSENSUS_NODE = 'true';
     });
 
-    it('eth_call with missing `to` field', async function () {
```

Comment:
Since now `to` field is allowed to be null/undefined, this test case is no longer relavant. Instead, this [test case here](https://github.com/hashgraph/hedera-json-rpc-relay/pull/2647/files#diff-5b9587c533d686b0386eb8c09b87a02ab40ec1b9581f0a471f03049465cd7698R746) cover the new support.